XMonad/Hooks/RefocusLast.hs

```diff
@@ -41,7 +41,9 @@ module XMonad.Hooks.RefocusLast (
   RecentWins(..),
   RecentsMap(..),
   RefocusLastLayoutHook(..),
-  RefocusLastToggle(..)
+  RefocusLastToggle(..),
+  -- * Library functions
+  withRecentsIn,
```

Comment:
I wonder whether it would be better to instead expose every utility here (they can all be recreated by what's already exported) or if we should do this on a case-by-case basis, depending on when they are needed

Reply:
I'm not sure why `RecentsMap` was exported in the first place, but your argument makes sense (although if I could, I'd un-export `RecentsMap`)

Reply:
Yes that would be the optimal solution but, alas, it has already made its way to a release.  We could think about deprecating the export (is that possible?  If not I guess a simple deprecation of the function that informs users about this would also work) but that's not something I necessarily want to do in this pr